test/src/org/omegat/util/StringUtilTest.java

```diff
@@ -428,4 +556,121 @@ public void testCompareToNullable() {
         assertEquals(32, StringUtil.compareToNullable("a", "A"));
         assertEquals(-32, StringUtil.compareToNullable("A", "a"));
     }
+
+    @Test
+    public void testReplaceSquaredLatinAbbreviations() {
```

Comment:
It is better to write functional test for `normalizeWidth` method for future replacement of the functionality with 3rd party library.